SDK-v4/gdpr/README.md

```diff
@@ -0,0 +1,27 @@
+# GDPR Compliant example
+
+Sample project demonstrating the integration of Localytics for a customer supporting GDPR on SDK versions below SDK v5.1.
+This app wraps a few Localytics calls in order to ensure that no data is sent to Localytics when a user is opted out of data collection. This code is meant as a sample in order to show off an example implementation, and is not intended as a complete solution. Specifically, this sample shows the integration of only a few Localytics calls: `tagEvent` and `setProfileAttribute`.  All additional Localytics API calls should be wrapped in a similar fashion. To ensure you have properly completed a GDPR compliant integration, please consult your Localytics Mobile Engagement Consultant before going live.
+
+## Prerequisites
+
+This app integrates against SDK v4.5.1.
+
+## Setup
+
+1. In `LocalyticsGDPRWrapper` replace `YOUR-LOCALYTICS-APP-KEY` with your Localytics app key.
+2. Build and run.
+
+## Summary
+This app is gated by an authentication screen.  It is required that the end user log into the app in order to use the underlying features.  As a result, we don't deal with anonymous users.
+
+## Limitations
+Because SDK v4 was not designed to uphold GDPR specifications, there are some potential concerns that customers may still have with this integration.
+
+- Attribution Source will forever be retained on the device, and reuploaded if a user opts back in.
+- First Ever session open datapoint will forever be retained on the device, and reuploaded if a user opts back in.
```

Comment:
First Ever session open datapoint will be retained indefinitely on the device, and re-uploaded if a user opts back in.